src/components/AppWrapper/AppWrapper.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "styled-components";
+
+import Header from "../Header";
+import MainContent from "../MainContent";
+import Footer from "../Footer";
+
+const StyledAppWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+`;
+
+const AppWrapper = () => {
+  return (
+    <StyledAppWrapper>
+      <Header />
+      <MainContent />
+      <Footer />
```

Comment:
It is noticeable in both cases (little or a lot of main content). Under the footer noticeable small space - appears when you scroll, could you fix it?

https://user-images.githubusercontent.com/48246024/224614646-e407469c-a3c0-413f-904f-822e674b1f50.mov